.github/workflows/coverage-orchestrator.yml

```
@@ -0,0 +1,26 @@
+    secrets: inherit
+
+  run-pigz:
+    uses: ./.github/workflows/pigz.yml
+    secrets: inherit
+
+  # This job only starts if ALL reusable workflows above succeed
+  final-upload:
+    needs: [run-cmake, run-lint]
+    runs-on: ubuntu-slim
+    if: |
+      success()
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)